nfts.proto

```diff
@@ -235,5 +241,6 @@ message NftEvents {
     UpdateSolanaMintPayload solana_updated_collection_mint = 41;
     RetryUpdateSolanaMintPayload solana_retry_updated_collection_mint = 42;
     SolanaUpdatedMintPayload solana_mint_updated = 43;
+    SwitchCollectionPayload solana_switch_mint_collection = 44;
```

Comment:
```suggestion
    SwitchCollectionPayload solana_switched_mint_collection = 44;
```

Reply:
going to update collection id for mint after the transaction is successful